build.sbt

```diff
@@ -22,7 +22,7 @@ lazy val commonSettings = Seq(
   organization := "com.cognite.spark.datasource",
   organizationName := "Cognite",
   organizationHomepage := Some(url("https://cognite.com")),
-  version := "1.4.54",
+  version := "1.4.55.SNAPSHOT.1",
```

Comment:
Use -SNAPSHOT to be able to publish snapshots to Sonatype

Reply:
done.